src/theme/variables/edgeLight.js

```diff
@@ -201,6 +202,9 @@ export const edgeLight: Theme = {
   underlayColor: palette.white,
   underlayOpacity: 0.95,
 
+  // Tutorials
+  walletListSlidingTutorialModalUnderlay: palette.transparent,
```

Comment:
This will be common for all tutorials. Rename to `tutorialModalUnderlay`

Reply:
fixed